test/errors/incompatibleKinds-detailed.good

```diff
@@ -0,0 +1,49 @@
+─── error in incompatibleKinds.chpl:3 [IncompatibleKinds] ───
+  A regular variable cannot be initialized with a type.
+  In the following initialization expression:
+      |
+    3 | var a = int;
+      |         ⎺⎺⎺  
+      |
+  the initialization expression is a type 'int(64)'.
+
+─── error in incompatibleKinds.chpl:4 [IncompatibleKinds] ───
+  A type variable cannot be initialized using a regular value.
+  In the following initialization expression:
+      |
+    4 | type b = notAType;
+      |          ⎺⎺⎺⎺⎺⎺⎺⎺  
+      |
+  the initialization expression is a value of type 'string'.
```

Comment:
I can see that, but personally I think "expression" doesn't exactly make it clear that it's _not_ a type. Perhaps "expression would result in a value of type 'string'", or something, to make it clear that it may not be a literal right now, but eventually would become a value?

Reply:
Hmm, that's a good point, since I agree that `myTypeComputingFn()` is an expression as well.  And in retrospect, I'm seeing that my suggestion would have led to "expression...expression" repetitiveness as well.  What about simply "the initialization expression is of type 'string'" or "...has type 'string'"?  Though I guess that doesn't make it clear enough that it isn't simply the type `string` itself.  OK, I'm more OK with leaving this as-is, then...  It still gives me slight pause, but not enough to wrestle with further at this point.